src/app/api/graphql/route.js

```diff
@@ -8,7 +8,7 @@ import { gql } from 'graphql-tag'
 import { MongoClient } from 'mongodb'
 
 // The connection string for mongodb connection.
-const uri = process.env.MONGODB_URI? process.env.MONGODB_URI : ''
+const uri = process.env.MONGODB_URI ? process.env.MONGODB_URI : ''
```

Comment:
It's not a necessity, but if you'd like you can use the `OR` operator here.

```javascript
const uri = process.env.MONGODB_URI || '';
```

Reply:
Done and commited in master branch!